src/components/carousel/index.js

```diff
@@ -367,6 +395,7 @@ export default class Carousel extends BaseComponent {
           onContentSizeChange={this.onContentSizeChange}
           onScroll={this.onScroll}
           onMomentumScrollEnd={this.onMomentumScrollEnd}
+          overflow={'visible'}
```

Comment:
I don't think overflow it's a prop but a style prop. 
you can remove this one

Reply:
You are right, removed